packages/editor/src/components/document-bar/index.js

```diff
@@ -184,11 +187,15 @@ export default function DocumentBar( props ) {
 									? decodeEntities( title )
 									: __( 'No title' ) }
 							</span>
-							{ postTypeLabel && ! props.title && (
-								<span className="editor-document-bar__post-type-label">
-									{ '· ' + decodeEntities( postTypeLabel ) }
-								</span>
-							) }
+							<span className="editor-document-bar__post-type-label">
```

Comment:
I'd suggest to avoid to render a potentially emptuy `<span>`. See in the previous implementaiton the condition wraps the entire span.

Reply:
Sure, I would revert that change.